cognite/data_objects.py

```diff
@@ -33,6 +34,34 @@ def to_ndarray(self):
         '''Returns data as a numpy array'''
         return self.to_pandas().values
 
+
+class RawRowDTO(object):
+    """DTO for a row in a raw database."""
```

Comment:
Denne klassen kan gjerne dokumenteres i litt større detalj. Hva er key og columns her?

http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html eksempeldokumentering for klasser.

Reply:
La til litt mer!